source/_integrations/alarm_control_panel.markdown

```diff
@@ -6,6 +6,7 @@ ha_category:
 ha_release: 0.7.3
 ha_quality_scale: internal
 ha_domain: alarm_control_panel
+ha_iot_class: Local Push
```

Comment:
This specific integration actually can't be used. It is what we call an Entity Component. It is used by other integration to build upon. This integration by itself, actually can't do anything.

And thus, does not have a IoT class

```suggestion
ha_iot_class: ~
```

Reply:
Gotcha!  Once I get back to my laptop I will make that change!  Thank you for the explanation!